pkg/security/secl/model/model_windows.go

```diff
@@ -88,3 +104,32 @@ type NetworkDeviceContext struct{}
 type ExtraFieldHandlers interface {
 	BaseExtraFieldHandlers
 }
+
+// FIM
+
+// CreateNewFileEvent defines file creation
+type CreateNewFileEvent struct {
+	File FileEvent `field:"file"`
+}
+
+// Registries
+
+// CreateRegistryKeyEvent defines registry key creation
+type CreateRegistryKeyEvent struct {
+	Registry RegistryEvent `field:"registry"`
```

Comment:
maybe you could embed the RegistryEvent so that it will remove the 'registry' node in the middle of SECL expression